frontend/src/pages/DevicesPage/DevicesPage.tsx

```diff
@@ -74,6 +75,7 @@ export const DevicesPage = connect(
 
     useEffect(() => {
       analytics.page('DevicesPage')
+      emit('interfaces')
```

Comment:
Remove

Reply:
I kept this one. I'd like to have it on the connections page, devices page and settings page. So we have a better chance of it getting updated when a user comes back to the app. If you have another idea for this let me know.